compute/hello-world.py

```diff
@@ -9,14 +9,22 @@ def hello_world():
     orchestrator = swan.resource(api_key=os.getenv("SWAN_API_KEY"), network='mainnet',service_name='Orchestrator')
 
     result = orchestrator.create_task(
-        app_repo_image='hello_world',
+        repo_uri='https://github.com/swanchain/awesome-swanchain/tree/main/hello_world',
         wallet_address=os.getenv("WALLET_ADDRESS"),
         private_key=os.getenv("PRIVATE_KEY"),
+        instance_type='C1ae.small',
+        auto_pay=True
     )
     task_uuid = result['id']
     task_info = orchestrator.get_deployment_info(task_uuid=task_uuid)
     print(task_info)
 
+    # Wait for the task to be deployed
+    time.sleep(30)
```

Comment:
i don't think 30s is enough, it can be a while loop to break, or async.

Reply:
https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/python/example_code/ec2/instance.py#L141

Sample

Reply:
Yes, I agree with you. 30 seconds is far from enough in most cases. for discussion:

- the intention of putting a `sleep` in the hello-world example is simply to convey to the user that we need to *wait* for a while before application urls are available
- for full-fledged examples, we can show users some different waiting strategies. for example, waiting for a certain interval, and if not available, early terminate the task (similar to the way Lagrange is using), etc
- I thought this repo would contain examples from beginners' to advanced levels

Reply:
The developer may just copy the code and put into production use, so i think it it better handle the waiting time as AWS does, however, we have three instance in this case, so we may need to return the URL one the instance is online, but may have 403